### PR TITLE
feat(app): add ceramic did-session pkh flag support and dependencies

### PR DESCRIPTION
**Pull request by zachferland**
Didnt test entire flow, but did run app and authenticate using did-session pkh/siwe

### CODE REVIEW

**Comment by zachferland** [comment]
probably should of added a commit for diff, but edited

changes included dids version change and yarn.lock updates 

looks like it may be working now

**Review by gdixon — APPROVED** [review]
This is looking good to me 🚀 Everything is working as expected 🙌 

I've also tested the changes @oed has made to `Tulons` (https://github.com/tulons/tulons-ts/pull/1) and that is all working as expected too.

So, once this is merged we need to push out changes in `Tulons` and `passport_reader.py` - both are ready to go but I'll hold off until this is in.

@shavinac - can you checkout this branch and confirm everything is looking good for you? 

Thank you @zachferland for getting us switched over 🙏

**Review by shavinac — APPROVED** [review]
LGTM, thanks for the PR!
We can merge it in on Monday 🙌

**Pull request merged**
